src/aura/STG_CMP_System/STG_CMP_System.cmp

```diff
@@ -14,10 +13,10 @@
             <aura:if isTrue="{!v.isView}">
                 <ui:outputText aura:id="recTypeName" value="{!v.accRecTypeName}"/>
             <aura:set attribute="else">
-                <ui:inputSelect aura:id="accRecTypes">
+                <ui:inputSelect aura:id="accRecTypes" change="{!c.onSelectChange}">
                     <aura:iteration items="{!v.accRecTypes}" var="recType">
-                        <ui:inputSelectOption label="{!recType.devName}" text="{!recType.id}" 
-                            value="{!v.accRecTypeName == recType.devName ? true : false}" />
+                        <ui:inputSelectOption label="{!recType.name}" text="{!recType.id}" 
+                            value="{!v.accRecTypeId == recType.id ? true : false}" />
```

Comment:
Just to standardize storing recordtype IDs in all settings. The change to using dev names was done recently (https://github.com/SalesforceFoundation/HEDAP/pull/226/), as an intermediate step.